code/modules/jobs/job_types/psychologist.dm

```diff
@@ -16,7 +17,9 @@
 	liver_traits = list(TRAIT_MEDICAL_METABOLISM)
 
 	display_order = JOB_DISPLAY_ORDER_PSYCHOLOGIST
-	departments = DEPARTMENT_SERVICE
+	departments_list = list(
+		/datum/job_department/service,
```

Comment:
shouldnt this be service + medical? or am i misunderstanding the system

Reply:
same with service + sec for lawyer

Reply:
When uomo implemented the psychologist they chose to make them a service job, and said we could move them if they got actual medical mechanics content or something.
Lawyers have been service since forever.
This doesn't change the existent behavior.

Adding new departments for them is viable but it should be done on its own PR, as this is a refactor and I'm trying to keep as much of the existing behavior as possible.
Just take note that it will account their playtime on that job for the purpose of unlocking the head of department job. You could get a HoS who only played lawyer before. Not that I mind too much.